canonical-web-and-design/dotrun
# v1.1.0: Re-enable strict mode

[This semwraplib.so fix](https://forum.snapcraft.io/t/python-multiprocessing-permission-denied-in-strictly-confined-snap/15518/4?u=nottrobin) from @jdstrand gets Python multiprocessing working again, and by extension gets black working.

I've also removed macOS instructions for now - the `multipass mount` mechanism doesn't work very well for a couple of reasons. Firstly, it leads to an extremely slow experience, but also, the multipass mount breaks all the time. Remove it for now, we'll add instructions for a different setup later on.

I also moved the files to a `src/` directory so we can avoid accidentally putting more files in the snap than we need.

QA
--

``` bash
snap install --candidate dotrun
cd ~/Projects/ubuntu.com  # Or any webteam python project
dotrun build
dotrun exec black --line-length 79 webapp
```

^ See black succeed
